fem/bilininteg.hpp

```diff
@@ -901,7 +901,7 @@ class MixedVectorMassIntegrator : public MixedVectorIntegrator
    MixedVectorMassIntegrator() { same_calc_shape = true; }
    MixedVectorMassIntegrator(Coefficient &q)
       : MixedVectorIntegrator(q) { same_calc_shape = true; }
-   MixedVectorMassIntegrator(VectorCoefficient &dq)
+   MixedVectorMassIntegrator(DiagonalMatrixCoefficient &dq)
```

Comment:
This constructor should be kept for backward compatibility.

This remark also applies to other places where old constructors are replaced by new ones instead of adding the new one and marking the old as deprecated.

Reply:
Since `DiagonalMatrixCoefficient` is just a `typedef` of `VectorCoefficient` (see coefficient.hpp:691), this is not actually a change. I checked the other constructors edited in this PR, and they all seem to be like this, so I don't think backward compatibility is broken.

Reply:
Ah, I see. All good then. 👍